pyproject.toml

```diff
@@ -50,7 +50,7 @@ documentation = "https://django-pgpubsub.readthedocs.io"
 [tool.poetry.dependencies]
 python = ">=3.7.0,<4"
 django = ">=2"
-django-pgtrigger = ">=2.4.0"
+django-pgtrigger = ">=4.6.0"
```

Comment:
The 2.4.0 does not work correctly with django 3, I didn't dig deep why. It works with vesion 4 but on django 3 triggers cannot be added to models.

Reply:
I'll need to try to remember to actually write release notes for the next django-pgpubsub release, which will include this change. Two things of note are this requirement bump and the fact that it will require a new migration for those upgrading.

Reply:
Does it make sense to have a release notes document and make sure that every PR contains a record in the upcoming release section?

Reply:
Yeah it definitely would. This is something that's supposed to come packages with the Opus10 CI, which I'm still hoping Wes has some time to help me with soon.